src/com/sibvisions/kitchensink/samples/containers/SplitPanelSample.java

```diff
@@ -58,10 +74,39 @@ public IPanel getContent()
 		splitPanelThird.setFirstComponent(splitPanelFirstSecond);
 		splitPanelThird.setSecondComponent(createCenteredLabel("Third", Tango.SKY_BLUE_1));
 		
+		IDataBook controlsBook = new MemDataBook();
+		controlsBook.getRowDefinition().addColumnDefinition(new ColumnDefinition("ORIENTATION", new BigDecimalDataType(createOrientationCellEditor())));

```

Comment:
You could rename the columns to match the name of the components, like `ORIENTATION_SPLITPANELFIRST`.